### ISSUE TITLE
Low contrast/Bad font rendering (WCAG fail)

### ISSUE THREAD

**Issue by adamgreenough** [mjmlio/mjml-app#310]
![Screenshot 2020-01-02 at 19 08 04](https://user-images.githubusercontent.com/6561655/71686505-43410b00-2d93-11ea-954d-f5d9753a4cfb.png)

The text on latest version of app and macOS Catalina 10.5.2 is nowhere near as usable as the screenshot in the README would suggest.

I think that the font rendering may have changed and it is now very very light making this unusable for me. The contrast is already too low on top of this. The high contrast theme does not help.

The shot in the readme looks both brighter and bolder.

![Screenshot 2020-01-02 at 19 11 08](https://user-images.githubusercontent.com/6561655/71686680-c5313400-2d93-11ea-942c-32211982111c.png)
![Screenshot 2020-01-02 at 19 11 46](https://user-images.githubusercontent.com/6561655/71686684-c6626100-2d93-11ea-8320-982e11c76afe.png)
![Screenshot 2020-01-02 at 19 11 30](https://user-images.githubusercontent.com/6561655/71686686-c82c2480-2d93-11ea-851f-88bf2c6bc65c.png)

The web version is far superior in this regard. Regrettably I am not skilled enough to provide a PR for this but I hope that someone can help us out with this. :)

![Screenshot 2020-01-02 at 19 15 24](https://user-images.githubusercontent.com/6561655/71686863-530d1f00-2d94-11ea-9fe2-0f36189169b2.png)

**Comment by iRyusa** [comment]
This should be available on 3.0.0 now !
